src/framegrab/cli/preview.py

```diff
@@ -0,0 +1,72 @@
+import shutil
+
+import ascii_magic
+import click
+import cv2
+import yaml
+from imgcat import imgcat
+from PIL import Image
+
+from framegrab import FrameGrabber
+
+
+def imgcat_preview(name: str, frame):
+    """Displays the given frame in the terminal using imgcat."""
+    imgcat(frame)
+
+
+def cv2_preview(name: str, frame):
+    """Displays the given frame in a cv2 window."""
+    cv2.imshow(name, frame)
+    key = cv2.waitKey(30)
```

Comment:
Currently this puts an image on the screen for a very brief time. It might be better to do key = cv2.waitKey(0); cv2.destroyAllWindows() instead. This will put the image on the screen until the user presses any key, and then it will destroy the window.

Reply:
Done.